packages/react-radio/src/index.ts

```diff
@@ -26,3 +26,4 @@ export {
 } from './Radio';
 export type { RadioProps, RadioSlots, RadioState, RadioOnChangeData } from './Radio';
 export { RadioGroupContext } from './contexts/RadioGroupContext';
+export { useRadioGroupContextValues } from './contexts/useRadioGroupContextValues';
```

Comment:
_nit:_ You could also add an index file under `./contexts` and export from only one location here.